test_home.py::HomeTest::test_home_page
--------------------------------------------------------------------
Last Page: https://practice.automationbro.com/
  Browser: chrome (103.0.5060.134 / headless)
   Driver: chromedriver (103.0.5060.134)
Timestamp: 1659274701  (Unix Timestamp)
     Date: Sunday, 31 July 2022
     Time: 7:08:21 PM  (India Standard Time, UTC+05:30)
--------------------------------------------------------------------
Traceback: File "C:\Users\shash\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "C:\Users\shash\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 549, in _callTestMethod
    method()
  File "C:\Users\shash\PycharmProjects\pythonSelenium\tests\test_home.py", line 20, in test_home_page
    self.assert_title("Practice E-Commerce Site – Automation Br")
  File "C:\Users\shash\PycharmProjects\pythonSelenium\venv\lib\site-packages\seleniumbase\fixtures\base_case.py", line 6136, in assert_title
    self.assertEqual(expected, actual, error % (expected, actual))
  File "C:\Users\shash\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 845, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\Users\shash\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 1226, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "C:\Users\shash\AppData\Local\Programs\Python\Python310\lib\unittest\case.py", line 675, in fail
    raise self.failureException(msg)
Exception: 'Practice E-Commerce Site – Automation Br' != 'Practice E-Commerce Site – Automation Bro'
- Practice E-Commerce Site – Automation Br
+ Practice E-Commerce Site – Automation Bro
?                                         +
 : Expected page title [Practice E-Commerce Site – Automation Br] does not match the actual title [Practice E-Commerce Site – Automation Bro]!